README.md

```diff
@@ -1,13 +1,43 @@
 <p align="center">
     <img src="https://user-images.githubusercontent.com/19979068/77257398-9cedfe80-6c39-11ea-890a-9167ffd1b374.png">
+    <br /><i>An all-encompasing bioinformatics tool for genome assembly and annotation projects</i><br>
 </p>
 
-# GenoMagic
+---
+
+
+### TABLE OF CONTENTS
+
+1. [About GenoMagic](#sec1) </br>
+2. [Software Architecture](#sec2)</br>
+3. [Installation](#sec3)</br>
+4. [Feedback and bug reports](#sec4)<br />
```

Comment:
There's no need for the empty `<a name="secn"></a>` as you can refer to the headers directly: https://stackoverflow.com/questions/11948245/markdown-to-create-pages-and-table-of-contents

Reply:
Fixed.  202ac8e